services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceControllerMultiClusterConfig.java

```diff
@@ -387,6 +387,10 @@ public boolean isRealTimeTopicVersioningEnabled() {
     return getCommonConfig().getRealTimeTopicVersioningEnabled();
   }
 
+  public boolean isDarkCluster() {
+    return getCommonConfig().isDarkCluster();
+  }
+
```

Comment:
Is it possible to skip this part completely? `getCommonConfig()` is a pitfall for cluster level configs.

Reply:
Yes, skipped.